metricbeat/module/kubernetes/pod/data.go

```diff
@@ -122,12 +122,22 @@ func eventMapping(content []byte, perfMetrics *util.PerfMetricsCache) ([]common.
 			podEvent.Put("memory.usage.node.pct", float64(usageMem)/nodeMem)
 		}
 
-		if coresLimit > 0 {
-			podEvent.Put("cpu.usage.limit.pct", float64(usageNanoCores)/1e9/coresLimit)
```

Comment:
This code is about the cpu pct, this shouldn't be removed. Then redundant one is the previous if.

Reply:
🤦 Fixed

Reply:
Thank you for diligently reviewing my changes

Reply:
No problem :slightly_smiling_face: 

The `if nodeCores > 0...` in the lines 117 to 119 should be removed too, as `memory.usage.node.pct` is calculated in the new code now.